src/sentry/shared_integrations/client/proxy.py

```diff
@@ -87,13 +87,15 @@ def __init__(
             self.proxy_url = self.base_url
 
     @control_silo_function
-    def authorize_request(self, prepared_request: PreparedRequest) -> PreparedRequest:
+    def authorize_request(
+        self, prepared_request: PreparedRequest, **kwargs: Any
+    ) -> PreparedRequest:
         """
         Used in the Control Silo to authorize all outgoing requests to the service provider.
         """
         return prepared_request
 
-    def finalize_request(self, prepared_request: PreparedRequest) -> PreparedRequest:
+    def finalize_request(self, prepared_request: PreparedRequest, **kwargs: Any) -> PreparedRequest:
```

Comment:
Are we using these `**kwargs` anymore? I don't believe so after your refactor of `prepare_headers` but I might have missed it.

Reply:
No. I forgot to revert these changes.